docs/onboard-openshift.md

```diff
@@ -0,0 +1,76 @@
+# Guide to onboard developers into Openshift
+
+A guide for developers and platform engineers to onboard into containers and OpenShift.
+
+## Introduction
+
+Platform Team set up the OpenShift clusters. Clusters are ready to allocate workloads. Developers want to migrate their applications, but there needs to be a current process to onboard them. This document provides a template that can be used to design your onboarding process or use it as it is.
+
+## Summary
+
+### Developer Team Guide
+
+The Developer Team processes to be ready to start their container journey.
+
+1. Developer Foundational Learning
+2. Application Assessment: Apply Twelve-Factor App for Containers.
+
+[A checklist for The Top Twelve-Factor App for Containers]()
```

Comment:
Missing link?

Reply:
Yes the link is missing in the original document. For now it's a starting document with a skeleton and a few pages to help the editor to add more content.